esp-bootloader-esp-idf/build.rs

```diff
@@ -16,38 +16,8 @@ fn main() {
     println!("cargo::rustc-env=ESP_BOOTLOADER_BUILD_DATE={build_date_formatted}");
 
     // emit config
-    generate_config(
-        "esp-bootloader-esp-idf",
-        &[
-            ConfigOption::new(
-                "mmu_page_size",
-                "ESP32-C2, ESP32-C6 and ESP32-H2 support configurable page sizes. \
-                This is currently only used to populate the app descriptor.",
-                "64k",
-            )
-            .constraint(Validator::Enumeration(vec![
-                String::from("8k"),
-                String::from("16k"),
-                String::from("32k"),
-                String::from("64k"),
-            ])), // .active(true) TODO we need to know the device here
-            ConfigOption::new(
-                "esp_idf_version",
-                "ESP-IDF version used in the application descriptor. Currently it's \
-                not checked by the bootloader.",
-                "0.0.0",
-            ),
-            ConfigOption::new(
-                "partition-table-offset",
-                "The address of partition table (by default 0x8000). Allows you to \
-                move the partition table, it gives more space for the bootloader. Note that the \
-                bootloader and app will both need to be compiled with the same \
-                PARTITION_TABLE_OFFSET value.",
-                0x8000,
-            )
-            .constraint(Validator::PositiveInteger),
-        ],
-        true,
-        true,
-    );
+    println!("cargo:rerun-if-changed=./esp_config.yml");
+    let cfg_yaml = std::fs::read_to_string("./esp_config.yml")
```

Comment:
Stupid question, but what if we `include_std` the yamls?

Reply:
Probably just a matter of taste 🤔